lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Since we don't have any ORA1 problems on the platform, there is no new courses that would request this data.  We've been doing ORA2 requests for a while but I think this is fine to remove.  We haven't had an ORA1 grade dump request in a long time.